### ISSUE TITLE
Iterating Maps with Freemarker returns methods too

### ISSUE THREAD

**Issue by blurbiness** [moqui/moqui-framework#431]
There is an issue with Freemarker which makes that using `#list` with maps returns methods and not only map entries. This is related to the default `BeanWrapper` used.
I have seen that in Moqui source files, such as `moqui-quasar/template/screen-macro/DefaultScreenMacros.qvt.ftl`, the method `.keySet()` is used as a workaround.
But this prevent us from just using `#list myMap as k,v`.

Would it be a problem if default wrapper were changed in `moqui/framework/src/main/groovy/org/moqui/impl/context/renderer/FtlTemplateRenderer.java#
makeFtlConfiguration`?

This is from official freemarker documentation:
https://freemarker.apache.org/docs/app_faq.html#faq_simple_map

>**16.  When I list the contents of a map (a hash) with ?keys/?values, I get the java.util.Map methods mixed with the real map entries. Of course, I only want to get the map entries.**
Certainly you are using pure BeansWrapper as your object wrapper (instead of the default, DefaultObjectWrapper), or a custom subclass of it, and the simpleMapWrapper property of that is left to false. Unfortunately, that's the default of BeansWrapper (for backward compatibility), so you have to explicitly set it to true where you instantiate it. Also, at least since 2.3.22, applications should just use DefaultObjectWrapper (with its incompatibleImprovements set to at least 2.3.22 - that's especially important if you are switching from pure BeansWrapper), which never had this problem."

**Comment by jonesde** [comment]
For kicks I just tested this (changing MNode.java and FtlTemplateRenderer.java) and it is as I suspected, though it's been a long time since I worked on anything related to the FTL object wrapping. The result is you get a huge pile of errors (even in just the framework tests, ie something like 'gradle load framework:test').

It might possible to do this but would require a number of changes to existing FTL templates in moqui-runtime and elsewhere because there are many that use direct method calls that get blocked by the wrapping the DefaultObjectWrapper does.
